src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1967,6 +1967,12 @@ public static IEnumerable<DValue<RecordValue>> StandardTableNodeRecords(IRContex
                 else if (!forceSingleColumn && arg is BlankValue bv && tableType.FieldNames.Count() > 1)
                 {
                     return DValue<RecordValue>.Of(bv);
+                }
+
+                // Handle the Unknown return type functions: Remove, Coelesce
+                else if (!forceSingleColumn && arg is BlankValue bv2 && !tableType.FieldNames.Any())

```

Comment:
>&& !tableType.FieldNames.Any() [](http://example.com/codeflow?start=69&length=30)

This can be merged into the previous if statement (`...arg is BlankValue bv && (fieldCount == 0 || fieldCount > 1))`)